src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -591,12 +590,12 @@ def test_orbit_from_horizons_has_expected_elements():
     # Taken from https://ssd.jpl.nasa.gov/horizons.cgi
     ss = Orbit.from_classical(
         Sun,
-        2.767107592216510 * u.au,
-        7.554803091400027e-2 * u.one,
-        2.718502494739172e1 * u.deg,
-        2.336913218336299e1 * u.deg,
-        1.322919809219236e2 * u.deg,
-        2.128957916690369e1 * u.deg,
+        2.76710759221651 * u.au,
```

Comment:
I was squinting at this and then realized that you just "moved the comma" :)

Reply:
Oh! Yeah, I thought it was more natural removing the scientific notation.